content/ui/admin/init.collage.js

```diff
@@ -156,7 +156,7 @@ GLOB.pers.init = function(){
         });
     
     $.concierge.addComponents({
-        notes_loader:    function(P, cb){GLOB.pers.call("getNotes", P, cb);}, 
+        notes_loader: function(P, cb){GLOB.pers.__components.notes_loader(P, cb);}, 
```

Comment:
christian says not---that GLOB.pers.__components.notes_loader is a *different* notes_loader function than the defined in concierge.addComponents.  GLOB.pers.__components.notes_loader is already defined to GLOB.pers.call("getNotes"), so having concierge's notes_loader invoke __components.notes_loader still invokes getNotes but now we've hidden away that implementation in pers.__components.